docs/templates/getting-started/sequential-model-guide.md

```diff
@@ -137,6 +137,22 @@ model.compile(optimizer='rmsprop',
 # for a mean squared error regression problem
 model.compile(optimizer='rmsprop',
               loss='mse')
+
+# for custom metrics
+def mean_pred(y_true, y_pred):
+    return K.mean(y_pred)
+
+def multi_metric(y_true, y_pred):
+    m1 = ...
+    m2 = ...
+    return {
+        'my_metric': m1,
+        'my_other_metric': m2
+    }
```

Comment:
In general I would prefer examples to be meaningful rather than dummy placeholders. It's a way to both explain usage patterns and convey a sense of what you can do. Also examples should include their import statements (`K` in this cas).